src/ui/DiffView.cpp

```diff
@@ -429,6 +429,7 @@ class DisclosureButton : public Button
 
 class Editor : public TextEditor
 {
+    Q_OBJECT
```

Comment:
Wrong indentation, same below. Please use two spaces like the existing code.

Reply:
I will fix it and submit again.